### PR TITLE
Every Database Provider has it own Reserved words.

### PR DESCRIPTION
**Pull request by jogibear9988**
Without the extra list for Postgres, our Querys do not work after switch from bltoolkit to linq2db

### CODE REVIEW

**Comment by jogibear9988** [comment]
Lists are from Migrator.NET, wich has them from the Database Manuals

File: Source/SqlQuery/ReservedWords.cs [hunk 1]
```diff
@@ -22,8 +22,38 @@ static ReservedWords()
...
+					ReservedWords.ReservedWordsDictionaryPostgres.Add(s, s);
+			}
+
+			name = assembly.GetManifestResourceNames().Single(_ => _.EndsWith("ReservedWordsOracle.txt"));
+#if NETFX_CORE
+			using (var stream = assembly.GetManifestResourceStream(name))
+#else
+			using (var stream = assembly.GetManifestResourceStream(name))
+#endif
+			using (var reader = new StreamReader(stream))
```
Comment (sdanyliv) [inline Source/SqlQuery/ReservedWords.cs]: Looks like preprocessor directives is not needed in this method.

**Comment by ili** [comment]
Why not to put all reserved words in one source?

**Comment by jogibear9988** [comment]
Because if you Quote a Reserved Word, it's Case Sensitive. And if you don't quote when creating the Column this could cause Problems. (It causes Problems in our code!). It works with BLToolkit, but not with Linq2DB any more!

**Comment by ili** [comment]
I see, i'll merge it on weekend

**Pull request merged**